example/storybook/stories/TrackTile/Indicator.stories.tsx

```diff
@@ -7,32 +7,35 @@ import Indicator, {
 import * as Icons from '@lifeomic/chromicons-native';
 import { kebabCase, invert } from 'lodash';
 import { View, Text, ScrollView } from 'react-native';
+import { CenterView } from '../../helpers/CenterView';
 
-storiesOf('Indicator', module).add('default', () => {
-  const names = Object.keys(Icons).map(kebabCase);
+storiesOf('Indicator', module)
+  .addDecorator((story) => <CenterView>{story()}</CenterView>)
```

Comment:
This is an example of a fixed story with the `CenterView` decorator.

| Before | After |
|-|-|
| ![Screenshot 2023-05-31 at 9 00 47 AM](https://github.com/lifeomic/react-native-sdk/assets/220893/15ddb979-5b06-444d-9bee-cf4f534b680b) | ![Screenshot 2023-05-31 at 9 01 43 AM](https://github.com/lifeomic/react-native-sdk/assets/220893/cdc80cb3-09a3-45d2-947e-964c185c644d) |